src/releng_docs/projects/shipit-static-analysis.rst

```diff
@@ -0,0 +1,203 @@
+          username: XXXX@mozilla.com
+          api_key: YYYYY
+
+
+The bot needs an environment variable ``MOZREVIEW`` with the following information:
+
+* the Mozreview mercurial revision of the patch to analyze (named ``<HG_SHA>`` here),
+* the Mozreview Review ID (named ``<MOZREVIEW_ID>`` here), 
+* the Mozreview Diff ID (named ``<MOZREVIEW_DIFF>`` here).
+
+So you'll need to do the following in the nix shell:
+
+.. code-block:: shell
+  
+  export MOZREVIEW="<HG_SHA>:<MOZREVIEW_ID>:<MOZREVIEW_DIFF>"
+
+Here is an example with this `Mozreview <https://reviewboard.mozilla.org/r/164530/>`_:
+
+1. You can get ``<MOZREVIEW_ID>`` straight from the url (``164530`` here)
+2. The Mercurial hash is in the first code sample (``hg pull -r ...``, so ``<HG_SHA>`` is : ``308c22e7899048467002de4ffb126cac0875c994``)
+3. To get the ``<MOZREVIEW_DIFF>``, click on the Diff tab, then you'll see the last diff of this patch (in this case it is diff 7)
+
+So the command would be:
+
+.. code-block:: shell
+  
+  export MOZREVIEW="308c22e7899048467002de4ffb126cac0875c994:164530:7"
+
+
+
+5. Setup a Phabricator test
+"""""""""""""""""""""""""""
+
+
+.. note::
+  Make sure your Taskcluster secret has a ``phabricator`` reporter setup, as follows (with a valid Phabricator uri & token):
+
+  .. code-block:: yaml
+
+    shipit-static-analysis:
+      ...
+      REPORTERS:
+        - reporter: phabricator
+          url: 'https://phabricator-dev.allizom.org/api/'
+          api_key: api-XXXX
+
+
+
+The bot needs an environment variable ``PHABRICATOR`` with the following information:
+
+1. The phabricator Differential ID (named ``DIFF_ID`` here)
+2. The phabricator Diff PHID (named ``PHID`` here)
+
+So you'll need to do the following in the nix shell:
+
+.. code-block:: shell
+  
+  export PHABRICATOR="<DIFF_ID>:<PHID>"
+
+Here is an example with this `Phabricator Diff review <https://phabricator-dev.allizom.org/D41>`_:
+
+1. You can get ``DIFF_ID`` from the url (this is ``41``)
+2. Login on the Phabricator instance (needed for API queries)
+3. Go to the Conduit API web interface (``/conduit`` of the Phabricator instance), and click on the endpoint ``differential.query`` (direct link to `Phabricator DEV <https://phabricator-dev.allizom.org/conduit/method/differential.query/>`_)
+4. Fill the form field ``ids`` as a JSON list of integer using ``DIFF_ID``, so for our example : ``[41]``
+5. Click ``Call Method``
+6. The method result should have a ``activeDiffPHID`` key, that's our ``PHID`` (in our example: ``PHID-DIFF-b5wsvctabxjmwqonwryv``)
+
+Here is the final command line:
+
+.. code-block:: shell
+  
+  export PHABRICATOR="41:PHID-DIFF-b5wsvctabxjmwqonwryv"
+
+
+6. Run the bot
+""""""""""""""
+
+Finally, you can run the bot with this command (in the Nix Shell):
+
+.. code-block:: shell
+
+  shipit-static-analysis \
+    --taskcluster-secret=repo:github.com/mozilla-releng/services:branch:master \
+    --cache-root=/app/tmp
+
+
+Test the bot in real conditions
```

Comment:
Maybe we should use the same title as `releng_tooltool` and `shipit_code_coverage` for consistency? (`Is ToolTool working correctly?`, `Is the code coverage project working correctly?`).